lab09/lab_09.cpp

```diff
@@ -27,19 +39,36 @@ int main()
   double fine_cost[MAX_PATRONS];
   int amount_of_records = 0;
 

```

Comment:
I'd probably add some docs describing what we want to do.  E.g.,

```cpp
// We want to load in an input file, calculate the fines,
// the ouptut a report detailing the overdue books by fines and days.
```

For future Ethan, that'll tell him what the intent of this code is.  Helpful 6 mos. down the road.